starter-code/mongoose-movies/models/movie.js

```diff
@@ -0,0 +1,16 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+// const Review = require('./review');
+
+const movieSchema = new Schema({
+    name: {
+        type: String,
+        required: [true, "Please input name"]
+    },
+    genre: String,
+    plot : String,
+    
+});
+
+const Product = mongoose.model('Movie', movieSchema);
+module.exports = Movie;
```

Comment:
Should have been exporting "Product" instead of "Movie" as that's how you defined it in line 15.